cmd/kubeless/deploy.go

```diff
@@ -122,11 +117,12 @@ var deployCmd = &cobra.Command{
 		if err != nil {
 			logrus.Fatal(err)
 		}
-
+		logrus.Infof("Creating function...")
```

Comment:
Tiny change: "Submitting function..." - to be consistent with the below info "Function submitted"